src/main/java/sic/modelo/SortingProducto.java

```diff
@@ -0,0 +1,35 @@
+package sic.modelo;
+
+public enum SortingProducto {
+    DESCRIPCION("descripcion"),
+    FECHA_ULTIMA_MODIFICACION("fechaUltimaModificacion"),
+    CODIGO("codigo"),
+    TOTAL_SUCURSALES("cantidadProducto.cantidadTotalEnSucursales"),
+    VENTA_POR_CANTIDAD("cantidadProducto.cantMinima"),
+    PRECIO_COSTO("precioProducto.precioCosto"),
+    PORCENTAJE_GANANCIA("precioProducto.gananciaPorcentaje"),
+    PRECIO_LISTA("precioProducto.precioLista"),
+    FECHA_ALTA("fechaAlta"),
+    PROVEEDOR("proveedor.razonSocial"),
+    RUBRO("rubro.nombre"),
+    ID_PRODUCTO("idProducto");
+
+    private final String nombre;
+
+    SortingProducto(String nombre) {
+        this.nombre = nombre;
+    }
+
+    public static SortingProducto fromValue(String value) {
+        for (SortingProducto sortingProducto : SortingProducto.values()) {
+            if (sortingProducto.nombre.equalsIgnoreCase(value)) {
+                return sortingProducto;
+            }
+        }
+        return null;
```

Comment:
Avoid return null

Reply:
This return is neccesary, the method fromValue must return a null value when there is no coicidence with "nombre", so the caller can know there is not a valid enum value for the given data

Reply:
I can change it for a empty String return